site/public/js/grade-inquiry.js

```diff
@@ -1,5 +1,25 @@
 /* global buildCourseUrl, WebSocketClient */
-/* exported initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
+/* exported saveDraft, initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
+
+function saveDraft() {
+    //saveDraft function displays the saved draft text on grade inquiry box
+    let draftContent = localStorage.getItem('content');
```

Comment:
`content` isn't a very descriptive key.  Additionally, won't it conflict with other drafts?

Reply:
Hi William, I renamed the "content" to "draftContent".